pylint/checkers/classes/class_checker.py

```diff
@@ -730,6 +730,12 @@ def _has_same_layout_slots(
         "Used when we detect that a property also has parameters, which are useless, "
         "given that properties cannot be called with additional arguments.",
     ),
+    "E0245": (
+        "No such name %r in __slots__",
```

Comment:
```suggestion
        "No such name '%r' in __slots__",
```

Reply:
I think this is going to create double simple quote ( "No such name ''c'' in __slots__") as the ``%r`` already call the repr function on a string.